app/views/admin_mailer/send_spam_alert.html.erb

```diff
@@ -0,0 +1,20 @@
+<% content_for :message do %>
+
+  <p>The following accounts have a suspicious level of traffic:</p>
+  <% @spam.each_pair do |user_id, info| %>
+    <% user = User.find(user_id) %>
+    User <%= style_link(user.login, root_url + user_works_path(user)) %> has a score of 
+    <%= info[:score] %>
+    <ul>
+      <% Work.where(id: info[:work_ids]).each do |work| %>
```

Comment:
Whoops, sorry, I deleted the comment I left on here before even though it was still relevant. 

The indenting and spacing here still needs to be fixed:

```
<% Work.where(id: info[:work_ids]).each do |work| %>
  <li>
    <%= style_link(work.title, work_url(work)) %>
    <% if work.spam? %>
      (<%= ts('Flagged as spam') %>)
    <% end %>
  </li>
<% end %>
```